src/main/java/picard/analysis/CollectInsertSizeMetrics.java

```diff
@@ -105,6 +105,9 @@
             "truncated to a shorter range of sizes, the MIN_HISTOGRAM_WIDTH will enforce a minimum range.", optional=true)
     public Integer MIN_HISTOGRAM_WIDTH = null;
 
+    @Argument(shortName = "TR", doc="Truncate the insert size histogram to throw out outliers", optional = true)
```

Comment:
I think it would be more user friendly to have an explicit argument to turn off truncating rather than requiring them to figure out that W=-1 means truncation. In the new commit I tried to strike the balance between the two approaches, let me know what you think.